### PR TITLE
Simple Code Refactor

### PR DESCRIPTION
**Pull request by interwebZninja**
A simple cfc refactored to cfscript... If this type of contribution is acceptable/desired, I can submit quite a few more. I've been doing it as an aid to understand how ColdBox works.

### CODE REVIEW

**Comment by lmajano** [comment]
I think this is great keep them coming, however, just make sure the tests for it are updated as well (if they exist).  Also, it just needs some of our new script syntax standards. I will commet inline. After that keep them coming.

File: system/logging/appenders/ConsoleAppender.cfc [hunk 1]
```diff
@@ -1,62 +1,48 @@
...
+ *  A simple ConsoleAppender
+ *
+ *  @author      Luis Majano
+ *  @createDate  04/12/2009
+ *
+ * *********************************************************************************** *
+ * Copyright Since 2005 ColdBox Framework by Luis Majano and Ortus Solutions, Corp     *
+ * www.coldbox.org | www.luismajano.com | www.ortussolutions.com                       *
+ * *********************************************************************************** */
+component extends="coldbox.system.logging.AbstractAppender" output="false" {
```
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: output is not necessary in script

File: system/logging/appenders/ConsoleAppender.cfc [hunk 2]
```diff
@@ -1,62 +1,48 @@
...
-	
-<!------------------------------------------- PRIVATE ------------------------------------------>
-	
-	
-</cfcomponent>
\ No newline at end of file
+ /**
+	*  Constructor
+	*
+	*  @name.hint        The unique name for this appender.
```
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: .hint is no longer necessary, railo fixed their bug

File: system/logging/appenders/ConsoleAppender.cfc [hunk 3]
```diff
@@ -1,62 +1,48 @@
...
+	*  @properties.hint  A map of configuration properties for the appender.
+	*  @layout.hint      The layout class to use in this appender for custom message rendering.
+	*  @levelMin.hint    The default log level for this appender, by default it is 0. Optional. ex: LogBox.logLevels.WARN
+	*  @levelMax.hint    The default log level for this appender, by default it is 4. Optional. ex: LogBox.logLevels.WARN
+	*/
+	public ConsoleAppender function init(required string name,
```
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: Can the arguments be in a separate line aligned like this:
```js
    required string name,
    struct properties={}
){

File: system/logging/appenders/ConsoleAppender.cfc [hunk 4]
```diff
@@ -1,62 +1,48 @@
...
+																								struct properties = {},
+																								string layout     = "",
+																								string levelMin   = 0,
+																								string levelMax   = 4 ) {
+		super.init(argumentCollection=arguments);
```
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: Please all ( and ) require spacing:
```
super.init( argumentCollection=arguments );
```

File: system/logging/appenders/ConsoleAppender.cfc [hunk 5]
```diff
@@ -1,62 +1,48 @@
...
+		return this;
+	}
+
+ /**
+	*  Write entry into appender
+	*
+	*  @logEvent.hint  The logging event.
+	*/
+	public void function logMessage(required any logEvent) {
+		if (hasCustomLayout()) {
```
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: Please update all spacing:
```
if( hasCustomLayout() ){
```

File: system/logging/appenders/ConsoleAppender.cfc (hunk 2 above)
Comment (interwebZninja) [inline system/logging/appenders/ConsoleAppender.cfc]: Understood... but Adobe still considers it "proper". Thoughts?

File: system/logging/appenders/ConsoleAppender.cfc (hunk 5 above)
Comment (interwebZninja) [inline system/logging/appenders/ConsoleAppender.cfc]: Sure... 
Forgive my ignorance, but what is the desired/standard indent rule? Tabs (Tab width 4)?

File: system/logging/appenders/ConsoleAppender.cfc (hunk 3 above)
Comment (interwebZninja) [inline system/logging/appenders/ConsoleAppender.cfc]: Is this the desired format even if there is only a single argument, or is it acceptable to keep single arguments on the same line?

File: system/logging/appenders/ConsoleAppender.cfc (hunk 5 above)
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: Yes, we use tab width 4

File: system/logging/appenders/ConsoleAppender.cfc (hunk 2 above)
Comment (lmajano) [inline system/logging/appenders/ConsoleAppender.cfc]: Can you point me to the docs on this one?

**Comment by lmajano** [comment]
Thanks, merged manually

**Pull request closed without merging**